Apps/ValidationTests/Win32/App.cpp

```diff
@@ -75,11 +74,13 @@ namespace
                 OutputDebugStringA(message);
             });
 
-            Babylon::Polyfills::Window::Initialize(env);
 
-            // Initialize NativeWindow plugin.
-            auto width = static_cast<float>(rect.right - rect.left);
-            auto height = static_cast<float>(rect.bottom - rect.top);
+            Babylon::Polyfills::Window::Initialize(env);
+            // Initialize NativeWindow plugin to the test size.
+            // TODO: TestUtils::UpdateSize should do it properly but the client size
```

Comment:
Maybe try to find out why? If not, then we should create an issue.

Reply:
https://github.com/BabylonJS/BabylonNative/issues/118